Tests/EndToEndTests/EndToEndTests.swift

```diff
@@ -4,18 +4,18 @@ import XCTest
 final class EndToEndTests: XCTestCase {
   func testCSS() throws {
     // Some of the APIs that we use below are available in macOS 10.13 and above.
-    guard #available(macOS 10.13, *) else {
-      return
-    }
-
     let process = Process()
     process.executableURL = productsDirectory.appendingPathComponent("swift-doc")
     process.arguments = ["generate", "--module-name", "SwiftDoc", "--format", "html", "Sources"]
 
     let pipe = Pipe()
     process.standardOutput = pipe
 
-    try process.run()
+    if #available(OSX 10.13, *) {
```

Comment:
Looks like I missed another macOS 10.13 change: https://github.com/SwiftDocOrg/swift-doc/runs/1215042760?check_suite_focus=true#step:5:67

Making that change now.